### ISSUE TITLE
Resarching Game Review Platforms and Brainstorming Requirements About the Game Review Feature of the Platform.

### ISSUE THREAD

**Issue by AliBasarann** [bounswe/bounswe2023group5#15]
Our platform is also a game review platform. Looking other game review platforms and getting ideas from them would be good for the platform. We can find the requirements that is not mentioned in the description more easily if we review the other platforms. I will be appending some of the game review platforms and their good features to the trello. Any game reviewing platforms that will be recommended are welcome.

**Comment by bilalatim** [comment]
* [Squawka](https://www.squawka.com/en/)
  * Squawka provides online and mobile football fans with amazing real time statistics and visualisations on the matches. We can also improve the user experience by adding visual statistics to our own system. Squawka analyses over 14 million on-ball actions in real time to display the performance of each player. We can also add things in our system where we can test the performance of players.
* [Kotaku](https://kotaku.com/)
  * This gaming reviews and news platform has a good design. There are also pages created by the players themselves. People like medium post game tips on their page. We can also allow players to create their own pages and share game tactics and tips.
 * [GameFaqs](https://gamefaqs.gamespot.com/)
   * On this platform, people can ask questions about the games. The more you help by answering other people's questions on this platform, the higher your score will be. We can also use this kind of scoring system.
 * [Ign](https://www.ign.com/)
   * This platform has a very sweet interface. It's a pretty cool platform where people can share their opinions about games, TV shows and movies. It can inspire us especially about design.
  @AliBasarann can you share your thoughts about these sites?

**Comment by AliBasarann** [comment]
I looked the websites that @bilalatim  mentioned, yet I could not enter the first website. The other websites are good and we can add their good properties to trello. I also want to say that the last website (Ign) have very good design. We can take this site as model while we are doing our website design.

**Comment by AliBasarann** [comment]
We researched game review platforms and we found websites that might be helpful. We also found some useful ideas and possible requirements about the game review feature of our platform under this issue. Thanks @bilalatim for his contributions.. The next step is determining requirements. I am closing the issue since we will determine possible requirements about game review feature of the platform under issue #23.
